src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,124 @@
+package com.epam.tc.hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.List;
+import java.util.Locale;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.Test;
+
+public class Exercise1Test {
+
+    @Test
+    public void exercise1Check() {
+
+        //setup Chrome Driver
+        System.setProperty("webdriver.chrome.driver", "/absolute/path/to/binary/chromedriver");
+        WebDriverManager.chromedriver().setup();
+
+        //setup options for Chrome Driver
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--no-sandbox", "start-maximized");
+
+        WebDriver driver = new ChromeDriver(options);
+
+        //step 1 - navigate to the site
+        driver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        SoftAssertions softAssertions = new SoftAssertions();
+
+        //step 2 - assert browser title
+        softAssertions.assertThat(driver.getTitle()).contains("Page Home");
+
+        //step 3 - perform login
+        WebElement element = driver.findElement(By.id("user-icon"));
+        element.click();
+
+        element = driver.findElement(By.id("name"));
+        element.click();
+        element.sendKeys("Roman");
+
+        element = driver.findElement(By.id("password"));
+        element.click();
+        element.sendKeys("Jdi1234");
+
+        element = driver.findElement(By.id("login-button"));
+        element.click();
+
+        //step 4 - Assert Username is loggined
+        driver.findElement(By.id("user-name"));
+        softAssertions.assertThat(driver.getTitle()).isEqualTo("Roman Iovlev");
+
+        //Assert that there are 4 items on the header section are displayed
```

Comment:
Till this point all the steps should be moved in Base class to avoid code duplicate. It's kind of precondition in your tests.